javascript/packages/orchestrator/src/providers/podman/podmanClient.ts

```diff
@@ -489,20 +489,32 @@ export class PodmanClient extends Client {
   }
 
   getPauseArgs(name: string): string[] {
-    return ["exec", name, "--", "bash", "-c", "echo pause > /tmp/zombiepipe"];
+    return [
+      "exec",
+      `${name}_pod-${name}`,
+      "bash",
+      "-c",
+      "echo pause > /tmp/zombiepipe",
+    ];
   }
   getResumeArgs(name: string): string[] {
-    return ["exec", name, "--", "bash", "-c", "echo resume > /tmp/zombiepipe"];
+    return [
+      "exec",
+      `${name}_pod-${name}`,
+      "bash",
+      "-c",
+      "echo resume > /tmp/zombiepipe",
+    ];
   }
 
   async restartNode(name: string, timeout: number | null): Promise<boolean> {
-    const args = ["exec", name, "--", "bash", "-c"];
+    const args = ["exec", `${name}_pod-${name}`, "bash", "-c"];
```

Comment:
why this change? Why not:
```suggestion
    const args = ["exec", `pod_${name}`, "bash", "-c"];
```

Reply:
Podman, at least in the version we have support, use `name_pod-name` as container identifier and that why we should use that way. I removed the `"--"` because that argument calls doesn't work in podman is from `k8s`. Since we never tested this particular fn in podman because should never work the were was it.